storage/plugins/oai-pmh-server/OaiPmhServerPlugin.php

```
@@ -0,0 +1,2410 @@
+            $xw->endElement(); // doc
+        }
+
+        $xw->endElement(); // mag
+    }
+
+    // ── Archival unit metadata (delegates to archives plugin formats) ─────────
+
+    /**
+     * @param array<string, mixed> $rec
+     */
+    private function writeArchivalUnitMetadata(\XMLWriter $xw, array $rec, string $metadataPrefix): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $xw. Configured minimum length is 3. (undefined)

(ShortVariable)